packages/dds/map/src/directory.ts

```diff
@@ -338,6 +338,8 @@ export class DirectoryFactory {
  * @sealed
  */
 export class SharedDirectory extends SharedObject<ISharedDirectoryEvents> implements ISharedDirectory {
```

Comment:
Are we adding IDisposed somewhere to make sure compiler helps us ensure signatures are correct?

Reply:
Added to implement IDisposable interface.